promql/functions.go

```diff
@@ -928,6 +937,7 @@ var FunctionCalls = map[string]FunctionCall{
 	"abs":                funcAbs,
 	"absent":             funcAbsent,
 	"absent_over_time":   funcAbsentOverTime,
+	"present_over_time":  funcPresentOverTime,
```

Comment:
This list is alphabetically sorted so far, so we should probably keep it like that.